tests/scenarios/backup_restore/run.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Copyright (C) 2019-2020 Siodb GmbH. All rights reserved.
+# Use of this source code is governed by a license that can be found
+# in the LICENSE file.
+
+## Global
+SCRIPT_DIR=$(dirname $(realpath "$0"))
+TEST_NAME=$(basename "${SCRIPT_DIR}")
+source "${SCRIPT_DIR}/../../share/CommonFunctions.sh"
+
+## Specific test functions
+
+## Specific test parameters
+SOURCE_DATA_DIR="/var/lib/siodb/siodb/data"
```

Comment:
there is variable SIODB_DATA_DIR set to `/var/lib/siodb/instance-name`, please use it to derive source and destination data directories